base/ca/src/main/java/org/dogtagpki/server/ca/CAEngine.java

```diff
@@ -1121,7 +1121,7 @@ public void initSubsystems() throws Exception {
 
             logger.info("CAEngine: Configuring OCSP responder");
 
-            ocspResponderByName = caConfig.getBoolean("byName", true);
+            ocspResponderByName = caConfig.getBoolean("byName", false);
```

Comment:
Wouldn't you want to flip the initialization to false  too?  If you do, I'll make a doc change request to the Admin's guide (small change).
protected boolean ocspResponderByName = true;

Reply:
I have removed the true to have the initialization to the default false